contracts/src/v0.7/tests/UpkeepCounter.sol

```diff
@@ -22,6 +22,7 @@ contract UpkeepCounter {
   }
 
   function performUpkeep(bytes calldata performData) external {
+    require(eligible(), "no longer eligible");
```

Comment:
I removed this check to avoid reverted transactions. 

```
The issue of failed txs is related to the thing, that AVAX does not have a block time.
It depends on the blockchain load: there can be too many blocks per small period of time, or no blocks at all.
For instance, there is an upkeep (0xb9faF0337db8238758570A51D0aF8B298861A14A) contract that have blocks test range 5915551 - 5916551.
The last performance was included into 5916529 block. The next performance is expected on 5916529 + 20 = 5916549 (2 blocks left before test range boundary).
The last check upkeep was at 2022-02-10T12:01:51.44 on blocks 5916551 (the last allowed one), and returned the green light (true) to perform an upkeep which is correct behavior.
Then the last upkeep performance transaction has been passed to the tx manager at 2022-02-10T12:01:51 and broadcasted a couple of seconds later on block 5916552 which is out of the test range blocks boundaries.
As a result, the transaction has been reverted because an upkeep is not eligible already.
```

Reply:
So I added this back after a conversation around double performs with Steve. But maybe there is still some confusion here on what makes or blocks that double behavior

Reply:
I'm not sure about this conversation and what was decided. This check will affect failed transactions, a lot. 
I had 1160 failed txs due to this check together with peculiarities of avax chain.

Reply:
ok i will remove it